README.md

```diff
@@ -2,31 +2,131 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Show one problem and how we can implement it with any of the loops we learned
+  about.
+- Discuss when certain loops should be used.
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+Consider the problem statement:
 
-## Topic 1
+Assume we want to print the integers from 0 to 5. So we'd expect the program
+to print numbers 0, 1, 2, 3, 4, 5.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+We can use any of the loops we have learned to accomplish this task. Consider
+the following program:
 
-## Topic 2
+```java
+public class LoopExample {
+  public static void main(String[] args) {
+      
+    // Implement a for loop that prints the integers 0 - 5
+    System.out.println("The For Loop prints:");
+    for (int i = 0; i <= 5; i++) {
+      System.out.println(i);
+    }
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+    // Implement a while loop that prints the integers 0 - 5
+    System.out.println("\nThe While Loop prints:");
+    int currentNumber = 0;
+    while (currentNumber <= 5) {
+      System.out.println(currentNumber);
+      currentNumber++;
+    }
 
-## Conclusion
+    // Implement a do-while loop that prints the integers 0 - 5
+    System.out.println("\nThe Do-While Loop prints:");
+    currentNumber = 0;
+    do {
+      System.out.println(currentNumber);
+      currentNumber++;
+    } while (currentNumber <= 5);
+
+  }
+}
+```
+
+The output of this program would look like this:
+
+```text
+The For Loop prints:
+0
+1
+2
+3
+4
+5
+
+The While Loop prints:
+0
+1
+2
+3
+4
+5
+
+The Do-While Loop prints:
+0
+1
+2
+3
+4
+5
+```
+
+As we can see by the output, each loop accomplishes exactly what the problem
+statement asks for; however, is there a loop that is more suited for this
+problem?
+
+In this lesson, we will learn about when to appropriately use each loop.
+
+## What Loop to Use and When
+
+Consider the following universal rules when using loops in programming:
 
-A short one or two paragraph summary of the contents of the lessons, recapping
-the learning goals.
+- Use a `for` loop when it is known how many iterations of the loop should run.
+- Use a `do-while` loop when the number of iterations known is at least once.
```

Comment:
Consider rewording, maybe something like 'when it is known that the loop should iterate at least once.'